create-release.py

```diff
@@ -36,7 +36,7 @@
 VERSION_REG_EX = r"(?P<major>\d+)\.(?P<minor>\d+)\.(?P<patch>\d+)(\.(?P<pre_release>[a-z]+)(?P<build>\d+))?"
 
 DEFAULT_GIT_URL = 'git@github.com:elyra-ai/elyra.git'
-DEFAULT_EXTENSION_PACKAGE_GIT_URL = 'git@github.com:elyra-ai/elyra-package-template.git'
+DEFAULT_GIT_BRANCH = ''
```

Comment:
```suggestion
DEFAULT_GIT_BRANCH = 'master'
```
Lets default this to master